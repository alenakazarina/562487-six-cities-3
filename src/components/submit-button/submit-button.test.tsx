import * as React from 'react';
import * as renderer from 'react-test-renderer';
import SubmitButton from './submit-button';

describe(`SubmitButton`, () => {
  it(`should render login SubmitButton`, () => {
    const tree = renderer.create(
        <SubmitButton
          prefix="login"
          isDisabled={false}
        />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it(`should render login SubmitButton disabled`, () => {
    const tree = renderer.create(
        <SubmitButton
          prefix="login"
          isDisabled={true}
        />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it(`should render reviews SubmitButton`, () => {
    const tree = renderer.create(
        <SubmitButton
          prefix="reviews"
          isDisabled={false}
        />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it(`should render reviews SubmitButton disabled`, () => {
    const tree = renderer.create(
        <SubmitButton
          prefix="reviews"
          isDisabled={true}
        />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
