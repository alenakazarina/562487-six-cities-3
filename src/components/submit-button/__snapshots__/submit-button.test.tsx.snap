// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubmitButton should render login SubmitButton 1`] = `
<button
  className="login__submit form__submit button"
  disabled={false}
  type="submit"
>
  Sign in
</button>
`;

exports[`SubmitButton should render login SubmitButton disabled 1`] = `
<button
  className="login__submit form__submit button"
  disabled={true}
  type="submit"
>
  Sign in
</button>
`;

exports[`SubmitButton should render reviews SubmitButton 1`] = `
<button
  className="reviews__submit form__submit button"
  disabled={false}
  type="submit"
>
  Submit
</button>
`;

exports[`SubmitButton should render reviews SubmitButton disabled 1`] = `
<button
  className="reviews__submit form__submit button"
  disabled={true}
  type="submit"
>
  Submit
</button>
`;
